.github/workflows/cd.yml

```diff
@@ -59,5 +59,46 @@ jobs:
         with:
           identifier: o2sh.onefetch
           installers-regex: '\.exe$'
-          version: ${{ github.event.release.tag_name }}
```

Comment:
Why was this line removed?

Reply:
It was added in 5085c5b038c46ecb9fb16329caf5479b35a963c6, and then removed again when I reverted that commit. Now, my intention was to revert everything *except the addition of this line*, which is why the commit message says “Revert "fix cd" (but keep the added version)”, but obviously I missed actually fixing up the commit. I’ll correct it. Thanks for catching this.